openwebstart/src/main/java/com/openwebstart/jvm/LocalRuntimeManager.java

```diff
@@ -308,7 +309,8 @@ public LocalJavaRuntime install(final RemoteJavaRuntime remoteRuntime, final Con
             }
         }
 
-        final LocalJavaRuntime newRuntime = new LocalJavaRuntime(remoteRuntime.getVersion(), remoteRuntime.getOperationSystem(), remoteRuntime.getVendor(), runtimePath);
+        final String vendor = VendorManager.getInstance().getInternalName(remoteRuntime.getVendor());
+        final LocalJavaRuntime newRuntime = new LocalJavaRuntime(remoteRuntime.getVersion(), remoteRuntime.getOperationSystem(), vendor, runtimePath);
```

Comment:
Could `LocalJavaRuntime` do the lookup using the `VendorManager`?
This would keep the code of the clients of `LocalJavaRuntime` simpler (no code duplication)